modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -72,7 +72,7 @@ def eval_insert(modin_df, pandas_df, **kwargs):
     eval_general(
         modin_df,
         pandas_df,
-        operation=lambda df, **kwargs: df.insert(**kwargs),
+        operation=lambda df, **kwargs: (df.insert(**kwargs), df)[1],
```

Comment:
What is the purpose for this change?

Reply:
`df.insert()` returns `None`, so `eval_general()` can not compare that inserts actually produced identical dataframes for pandas and Modin; this change makes the `lambda` to return modified `df`, so `eval_general()` will compare the results properly

Reply:
@vnlitvinov I think you can pass `__inplace__=True` instead: https://github.com/modin-project/modin/blob/5086a9ea37bc37e6e58da0ceaf5864b16cc8e0ed/modin/pandas/test/utils.py#L710

Reply:
will use, thanks! I've missed that argument...

Reply:
reverted my change, that test already had `__inplace__=True` argument